src/components/Header/NameExiticon.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { useSelector, useDispatch } from "react-redux"
+import { NavLink } from 'react-router-dom'
+
+import { fetchLogout } from "../../ducks/userInfo"
+import {
+    ExitIconWrapper,
+    MenuInitiaLetterCircle,
+    MenuInitiaLetter,
+    MenuName
+} from "./Header.styled"
+import ExitIcon from "../../assets/image/Header-exit-icon.svg"
+
+const NameExiticon = () => {
```

Comment:
I can understand from the name what does this component do

Reply:
Component renamed to - Logauticon

Reply:
Component renamed to - Logouticon

Reply:
LogoutIcon)